packages/trader/src/AppV2/Components/Guide/guide.scss

```diff
@@ -191,21 +191,39 @@
 
 .modal-player {
     position: fixed;
+    inset: 0;
     padding: 0;
     border: unset;
     overflow: hidden;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: var(--core-color-solid-slate-1400);
+    z-index: 999;
 
-    &::backdrop {
-        background-color: var(--core-color-opacity-black-600);
+    &__container {
+        height: 100%;
+    }
+
+    &__icon {
+        &-close {
+            position: fixed;
+            top: 0;
+            right: 0;
```

Comment:
@henry-deriv will this work for rtl screens. if not try this 
inset-inline-start: 0;

Reply:
im not familiar with close button on youtube for arabic language. Do they put it on the left or right